jdbc-perf-logger-driver/src/main/java/ch/sla/jdbcperflogger/logger/PerfLogger.java

```diff
@@ -275,4 +276,45 @@ static String getValueAsString(final SqlTypedValue sqlTypedValue, final Database
         strBuilder.append("*/");
         return strBuilder.toString();
     }
+    public static String removeComments(String result){
```

Comment:
I changed the code to manage
1. Multiline comments --comment1\n--comment2
2. Mutiline block comments /*first\n second*/
3. the concatenation is already working, because 
        
> if  (sqlStatement.charAt(i) == '\'') {
                    inString = !inString;
                }
                if (!inString) {
                  //nothing happens here because we check for comments to parse
                }
                if (i < sqlStatement.length() && !isCarriageReturnFromComment) {
                    cleanedStatement.append(sqlStatement.charAt(i));
                }